src/Exception/AuthorizationException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Joindin\Api\Exception;
+
+use \Exception;
+use \Throwable;
+
+final class AuthorizationException extends Exception implements Throwable
+{
+    private const MESSAGE = 'This operation requires %s privileges.';
+
+    public static function forNonAdministrator(): self
+    {
+        return new static(sprintf(self::MESSAGE, 'admin'), 403);
```

Comment:
What do you think about using the corresponding constant from `teapot/status-code`?

```diff
diff --git a/src/Exception/AuthorizationException.php b/src/Exception/AuthorizationException.php
index 6e3b6e8..e22ea7d 100644
--- a/src/Exception/AuthorizationException.php
+++ b/src/Exception/AuthorizationException.php
@@ -5,6 +5,7 @@ declare(strict_types=1);
 namespace Joindin\Api\Exception;

 use \Exception;
+use Teapot\StatusCode\Http;
 use \Throwable;

 final class AuthorizationException extends Exception implements Throwable
@@ -13,6 +14,6 @@ final class AuthorizationException extends Exception implements Throwable

     public static function forNonAdministrator(): self
     {
-        return new static(sprintf(self::MESSAGE, 'admin'), 403);
+        return new static(sprintf(self::MESSAGE, 'admin'), Http::FORBIDDEN);
     }
 }
```